e2e-tests/cypress/tests/integration/channels/enterprise/system_console/channel_moderation/create_posts_spec.ts

```diff
@@ -92,7 +92,7 @@ describe('MM-23102 - Channel Moderation - Create Posts', () => {
 
         // # Check Member should not have the permission to create a post on a channel when the option is removed.
         // * User should see a message stating that this channel is read-only and the textbox area should be disabled
-        cy.findByTestId('post_textbox_placeholder').should('have.text', 'This channel is read-only. Only members with permission can post here.');
+        cy.findByTestId('post_textbox').should('have.text', 'placeholder', 'This channel is read-only. Only members with permission can post here.');
```

Comment:
This is causing the failure in the last E2E run (https://automation-dashboard.vercel.app/cycle/12650852048_1-dfb076b-pr-onprem-ent). I think this might be a typo since I wouldn't expect there to be 3 arguments passed to this